docs/modules/deploy/pages/versioning-compatibility.adoc

```diff
@@ -60,13 +78,7 @@ Hazelcast Platform has been tested against the following operating systems.
 |Ubuntu
 |✓
 
-|Solarisfootnote:one-off[Tested on demand and not 100% guaranteed (One-off, commercially-reasonable efforts support).]
-|✓
-
-|IBM zOSfootnote:one-off[]
-|✓
-
-|AIX 7.2footnote:one-off[]
+|IBM zOSfootnote:one-off[Tested on demand and not 100% guaranteed (One-off, commercially-reasonable efforts support).]
```

Comment:
IBM zOS is supported only via a specific commercial agreement, not for all users / customers.

Reply:
Updated as requested.